docs/docs/platform/preferences.md

```diff
@@ -2,31 +2,208 @@
 sidebar_position: 7
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Subscriber Preferences
 
 Novu provides a way to store user preferences in the subscribers data model.
 This allows subscribers to specify and manage their preferences, without your intervention. Customizing preferences has become the standard expected behavior for people and Novu can take the technical burden of managing preferences off of you.
 
-## Managing subscriber preferences defaults
+Novu allows two levels of preferences:-
+
+- Template level channel preferences
+- Subscriber level channel subscription preferences
```

Comment:
> Subscriber level channel subscription preferences

What does the `subscription` here refer to though?

Reply:
Never mind. i know what it means. Just felt weird initially

Reply:
Maybe we can do `Template-level` and `Subscriber-level`

Reply:
subscribing to particular channel

Reply:
Done